test-utils/Cargo.toml

```diff
@@ -32,6 +32,7 @@ processor = { package = "miden-processor", path = "../processor", version = "0.1
     "testing",
 ] }
 prover = { package = "miden-prover", path = "../prover", version = "0.10", default-features = false }
+stdlib = { package = "miden-stdlib", path = "../stdlib", version = "0.10", default-features = false }
```

Comment:
It seems like `test-utils` depends on `stdlib`, and `stdlib` depends on `test-utils`. What am I missing? Why isn't this an error?

Reply:
Why do we need the `stdlib` dependency here? I would prover not to introduce it (if we can avoid it).

Reply:
One way to solve this is to just manually copy the implementation of `sys::truncate_stack` into `test_utils` crate somewhere. Then, test program can be defined like:
```
{TRUNCATE_STACK_PROC} begin {} exec.truncate_stack end
```

Reply:
If I get it right, the problem is that usage of the constant inside the macro require this constant to be available in the modules where this macro is used. So in order to create a program using the source code of the `truncate_stack` procedure I need to manually paste in inside the macro (which will be ugly, but will work)